# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Adam Pribyl <pribyl@lowlevel.cz>, 2008-2009
# Adam Přibyl <pribyl@lowlevel.cz>, 2008
# Adam Slovacek <adamslovacek@gmail.com>, 2013
# Filip Hruska <fhr@fhrnet.eu>, 2013
# Filip Hruska <fhr@fhrnet.eu>, 2015
# FooBar <thewired@riseup.net>, 2015
# Jan Splíchal <splichal.jan@gmail.com>, 2014
# Jiří Vírava <appukonrad@gmail.com>, 2014-2015,2017-2018
# dope <maniak11@gmail.com>, 2013
# Michal Várady <miko.vaji@gmail.com>, 2015
# Michal Vašíček <michalvasicek@icloud.com>, 2017
# Mikulas Holy, 2017
# Ondrej Führer <ondrej.fuhrer@gmail.com>, 2018
# Radek Bensch <inactive+Radog@transifex.com>, 2013
msgid ""
msgstr ""
"Project-Id-Version: Tor Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-10-20 12:34+0200\n"
"PO-Revision-Date: 2018-11-01 10:27+0000\n"
"Last-Translator: Jiří Vírava <appukonrad@gmail.com>\n"
"Language-Team: Czech (http://www.transifex.com/otf/torproject/language/cs/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: cs\n"
"Plural-Forms: nplurals=4; plural=(n == 1 && n % 1 == 0) ? 0 : (n >= 2 && n <= 4 && n % 1 == 0) ? 1: (n % 1 != 0 ) ? 2 : 3;\n"

#: ../tails_installer/creator.py:100
msgid "You must run this application as root"
msgstr "Tuto aplikaci musíte spouštět jako root"

#: ../tails_installer/creator.py:146
msgid "Extracting live image to the target device..."
msgstr "Rozbalování obrazu na cílové zařízení..."

#: ../tails_installer/creator.py:153
#, python-format
msgid "Wrote to device at %(speed)d MB/sec"
msgstr "Zapsáno na zařízení %(speed)d MB/sec"

#: ../tails_installer/creator.py:296
#, python-format
msgid ""
"There was a problem executing the following command: `%(command)s`.\n"
"A more detailed error log has been written to '%(filename)s'."
msgstr "Vyskytl se problém s vykonáním následujícího příkazu: '%(command)s'\nPodrobnosti o této chybě byly zapsány do souboru '%(filename)s'."

#: ../tails_installer/creator.py:315
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "Ověřuji kontrolní součet SHA1 obrazu LiveCD..."

#: ../tails_installer/creator.py:319
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "Ověřuji kontrolní součet SHA256 obrazu LiveCD..."

#: ../tails_installer/creator.py:335
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr "Chyba: SHA1 vašeho LiveCD je neplatný. Tento program můžete spustit s parametrem --noverify abyste tuto kontrolu přeskočili."

#: ../tails_installer/creator.py:341
msgid "Unknown ISO, skipping checksum verification"
msgstr "Neznámé ISO, přeskakuji ověření kontrolního součtu"

#: ../tails_installer/creator.py:353
#, python-format
msgid ""
"Not enough free space on device.\n"
"%dMB ISO + %dMB overlay > %dMB free space"
msgstr "Není dostatek místa na zařízení.\n%dMB ISO + %dMB trvalé uložiště > %dMB volné místo"

#: ../tails_installer/creator.py:360
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "Vytvářím %sMB překrytí trvalého úložiště"

#: ../tails_installer/creator.py:421
#, python-format
msgid "Unable to copy %(infile)s to %(outfile)s: %(message)s"
msgstr "Nemohu zkopírovat %(infile)s do %(outfile)s: %(message)s"

#: ../tails_installer/creator.py:435
msgid "Removing existing Live OS"
msgstr "Odebírám existující Live OS"

#: ../tails_installer/creator.py:444 ../tails_installer/creator.py:457
#, python-format
msgid "Unable to chmod %(file)s: %(message)s"
msgstr "Nemohu změnit oprávnění %(file)s: %(message)s"

#: ../tails_installer/creator.py:450
#, python-format
msgid "Unable to remove file from previous LiveOS: %(message)s"
msgstr "Nemohu odebrat soubor z předchozího LiveOS: %(message)s"

#: ../tails_installer/creator.py:464
#, python-format
msgid "Unable to remove directory from previous LiveOS: %(message)s"
msgstr "Nemohu odebrat složku z předchozího LiveOS: %(message)s"

#: ../tails_installer/creator.py:512
#, python-format
msgid "Cannot find device %s"
msgstr "Nemohu nalézt zařízení %s"

#: ../tails_installer/creator.py:713
#, python-format
msgid "Unable to write on %(device)s, skipping."
msgstr "Nemohu zapisovat na %(device)s, přeskakuji."

#: ../tails_installer/creator.py:743
#, python-format
msgid ""
"Some partitions of the target device %(device)s are mounted. They will be "
"unmounted before starting the installation process."
msgstr "Některé oddíly, které jsou připojeny na cílovém zařízení %(device)s, budou před zahájením procesu instalace odpojeny."

#: ../tails_installer/creator.py:786 ../tails_installer/creator.py:1010
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr "Neznámý souborový systém. Vaše zařízení bude nutné zformátovat."

#: ../tails_installer/creator.py:789 ../tails_installer/creator.py:1013
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Nepodporovaný systém souborů: %s"

#: ../tails_installer/creator.py:807
#, python-format
msgid "Unknown GLib exception while trying to mount device: %(message)s"
msgstr "Neznámá výjimka GLib při pokusu o připojení zařízení: %(message)s"

#: ../tails_installer/creator.py:812
#, python-format
msgid "Unable to mount device: %(message)s"
msgstr "Nemohu připojit zařízení: %(message)s"

#: ../tails_installer/creator.py:817
msgid "No mount points found"
msgstr "Nenalezeny žádné body připojení"

#: ../tails_installer/creator.py:828
#, python-format
msgid "Entering unmount_device for '%(device)s'"
msgstr "Spouští se unmount_device pro '%(device)s'"

#: ../tails_installer/creator.py:838
#, python-format
msgid "Unmounting mounted filesystems on '%(device)s'"
msgstr "Odpojuje se připojený souborový systém na '%(device)s'"

#: ../tails_installer/creator.py:842
#, python-format
msgid "Unmounting '%(udi)s' on '%(device)s'"
msgstr "Odpojuje se '%(udi)s' z '%(device)s'"

#: ../tails_installer/creator.py:853
#, python-format
msgid "Mount %s exists after unmounting"
msgstr "Připojení %s existuje po odpojení"

#: ../tails_installer/creator.py:866
#, python-format
msgid "Partitioning device %(device)s"
msgstr "Vytváří se oddíly na zařízení %(device)s"

#: ../tails_installer/creator.py:995
#, python-format
msgid "Unsupported device '%(device)s', please report a bug."
msgstr "Nepodporované zařízení  '%(device)s', prosím nahlaste chybu"

#: ../tails_installer/creator.py:998
msgid "Trying to continue anyway."
msgstr "Snažím se pokračovat dál."

#: ../tails_installer/creator.py:1007 ../tails_installer/creator.py:1405
msgid "Verifying filesystem..."
msgstr "Ověřuji systém souborů..."

#: ../tails_installer/creator.py:1031
#, python-format
msgid "Unable to change volume label: %(message)s"
msgstr "Není možné změnit jmenovku svazku: %(message)s"

#: ../tails_installer/creator.py:1037 ../tails_installer/creator.py:1440
msgid "Installing bootloader..."
msgstr "Instaluji bootloader..."

#: ../tails_installer/creator.py:1064
#, python-format
msgid "Could not find the '%s' COM32 module"
msgstr "Nelze najít '%s' COM32 modul"

#: ../tails_installer/creator.py:1072 ../tails_installer/creator.py:1458
#, python-format
msgid "Removing %(file)s"
msgstr "Odebírám %(file)s"

#: ../tails_installer/creator.py:1186
#, python-format
msgid "%s already bootable"
msgstr "%s je bootovatelné"

#: ../tails_installer/creator.py:1206
msgid "Unable to find partition"
msgstr "Nemohu najít oddíl"

#: ../tails_installer/creator.py:1229
#, python-format
msgid "Formatting %(device)s as FAT32"
msgstr "Formátuji %(device)s jako FAT32"

#: ../tails_installer/creator.py:1289
msgid "Could not find syslinux' gptmbr.bin"
msgstr "Nelze najít syslinux gptmbr.bin"

#: ../tails_installer/creator.py:1302
#, python-format
msgid "Reading extracted MBR from %s"
msgstr "Čtení extrahované MBR z %s"

#: ../tails_installer/creator.py:1306
#, python-format
msgid "Could not read the extracted MBR from %(path)s"
msgstr "Nelze načíst extrahované MBR z %(path)s"

#: ../tails_installer/creator.py:1319 ../tails_installer/creator.py:1320
#, python-format
msgid "Resetting Master Boot Record of %s"
msgstr "Resetuji hlavní bootovací záznam %s"

#: ../tails_installer/creator.py:1325
msgid "Drive is a loopback, skipping MBR reset"
msgstr "Zařízení je smyčka, přeskakuji resetování hlavního bootovacího záznamu"

#: ../tails_installer/creator.py:1329 ../tails_installer/creator.py:1589
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "Kalkuluji SHA1 %s"

#: ../tails_installer/creator.py:1354
msgid "Synchronizing data on disk..."
msgstr "Synchronizuji data na disk..."

#: ../tails_installer/creator.py:1397
msgid "Error probing device"
msgstr "Chyba při snímání zařízení"

#: ../tails_installer/creator.py:1399
msgid "Unable to find any supported device"
msgstr "Nenalezeno žádné podporované zařízení."

#: ../tails_installer/creator.py:1409
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "Ujistěte se, že váš USB disk je připojen a naformátován na systéme souborů FAT"

#: ../tails_installer/creator.py:1412
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr "Nepodporovaný systém souborů: %s\nProsím proveďte zálohu a naformátujte USB disk systémem souborů FAT."

#: ../tails_installer/creator.py:1481
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr "Nemohu získat Win32_LogicalDisk; win32com výsledky"

#: ../tails_installer/creator.py:1536
msgid "Cannot find"
msgstr "Nemohu nalézt"

#: ../tails_installer/creator.py:1537
msgid ""
"Make sure to extract the entire tails-installer zip file before running this"
" program."
msgstr "Ujistěte se, že extrahujete celý tails-installer zip před spuštěním tohoto programu."

#: ../tails_installer/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "Neznámé vydání: %s"

#: ../tails_installer/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "Stahuji %s..."

#: ../tails_installer/gui.py:213
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr "Chyba: Nelze nastavit jmenovku nebo získat UUID vašeho zařízení. Nemohu pokračovat."

#: ../tails_installer/gui.py:260
#, python-format
msgid "Installation complete! (%s)"
msgstr "Instalace kompletní! (%s)"

#: ../tails_installer/gui.py:265
msgid "Tails installation failed!"
msgstr "Instalace Tails selhala"

#: ../tails_installer/gui.py:361
msgid ""
"Warning: This tool needs to be run as an Administrator. To do this, right "
"click on the icon and open the Properties. Under the Compatibility tab, "
"check the \"Run this program as an administrator\" box."
msgstr "Upozornění: Tento nástroj je nutné spustit jako Administrátor. Učiníte tak kliknutím pravým tlačítkem na ikonu a vyberte Vlastnosti. V záložce Kompatibilita vyberte možnost Spustit tento program jako správce."

#: ../tails_installer/gui.py:373
msgid "Tails Installer"
msgstr "Instalátor Tails"

#: ../tails_installer/gui.py:423 ../data/tails-installer.ui.h:2
msgid "Clone the current Tails"
msgstr "Klonovat současné Tails"

#: ../tails_installer/gui.py:430 ../data/tails-installer.ui.h:3
msgid "Use a downloaded Tails ISO image"
msgstr "Použít stažený ISO obraz Tails"

#: ../tails_installer/gui.py:467 ../tails_installer/gui.py:788
msgid "Upgrade"
msgstr "Aktualizovat"

#: ../tails_installer/gui.py:469
msgid "Manual Upgrade Instructions"
msgstr ""

#: ../tails_installer/gui.py:471
msgid "https://tails.boum.org/upgrade/"
msgstr "https://tails.boum.org/upgrade/"

#: ../tails_installer/gui.py:479 ../tails_installer/gui.py:700
#: ../tails_installer/gui.py:765 ../data/tails-installer.ui.h:7
msgid "Install"
msgstr "Instalovat"

#: ../tails_installer/gui.py:482 ../data/tails-installer.ui.h:1
msgid "Installation Instructions"
msgstr "Instalační instrukce"

#: ../tails_installer/gui.py:484
msgid "https://tails.boum.org/install/"
msgstr "https://tails.boum.org/install/"

#: ../tails_installer/gui.py:490
#, python-format
msgid "%(size)s %(vendor)s %(model)s device (%(device)s)"
msgstr "%(size)s %(vendor)s %(model)s zařízení %(device)s)"

#: ../tails_installer/gui.py:502
msgid "No ISO image selected"
msgstr "Nebyl vybrán obraz ISO"

#: ../tails_installer/gui.py:503
msgid "Please select a Tails ISO image."
msgstr "Vyberte prosím obraz ISO s Tails."

#: ../tails_installer/gui.py:545
msgid "No device suitable to install Tails could be found"
msgstr "Nebylo nalezeno žádné zařízení vhodné pro instalaci Tails"

#: ../tails_installer/gui.py:547
#, python-format
msgid "Please plug a USB flash drive or SD card of at least %0.1f GB."
msgstr "Připojte prosím USB flash disk nebo SD kartu velikosti alespoň %0.1f GB."

#: ../tails_installer/gui.py:581
#, python-format
msgid ""
"The USB stick \"%(pretty_name)s\" is configured as non-removable by its "
"manufacturer and Tails will fail to start on it. Please try installing on a "
"different model."
msgstr "USB flash disk „%(pretty_name)s“ je nastaven jako neodebratelný jeho výrobcem a Tails se na něm nepodaří nastartovat. Zkuste prosím instalaci na jiný model."

#: ../tails_installer/gui.py:591
#, python-format
msgid ""
"The device \"%(pretty_name)s\" is too small to install Tails (at least "
"%(size)s GB is required)."
msgstr "Zařízení \"%(pretty_name)s\" je moc malé pro instalaci Tails (je vyžadováno alespoň %(size)s GB)."

#: ../tails_installer/gui.py:604
#, python-format
msgid ""
"To upgrade device \"%(pretty_name)s\" from this Tails, you need to use a downloaded Tails ISO image:\n"
"https://tails.boum.org/install/download"
msgstr "Pro aktualizaci zařízení \"%(pretty_name)s\" z tohoto Tails systému je nutno použít stažený Tails ISO obraz:\nhttps://tails.boum.org/install/download"

#: ../tails_installer/gui.py:625
msgid "An error happened while installing Tails"
msgstr "Při instalaci Tails nastala chyba"

#: ../tails_installer/gui.py:637
msgid "Refreshing releases..."
msgstr "Obnovování verzí..."

#: ../tails_installer/gui.py:642
msgid "Releases updated!"
msgstr "Verze obnoveny!"

#: ../tails_installer/gui.py:695
msgid "Installation complete!"
msgstr "Instalace kompletní!"

#: ../tails_installer/gui.py:711
msgid "Cancel"
msgstr "Zrušit"

#: ../tails_installer/gui.py:747
msgid "Unable to mount device"
msgstr "Nemohu připojit zařízení"

#: ../tails_installer/gui.py:754 ../tails_installer/gui.py:787
msgid "Confirm the target USB stick"
msgstr "Potvrďte cílový USB flash disk"

#: ../tails_installer/gui.py:755
#, python-format
msgid ""
"%(size)s %(vendor)s %(model)s device (%(device)s)\n"
"\n"
"All data on this USB stick will be lost."
msgstr "%(size)s %(vendor)s %(model)s zařízení (%(device)s)\n\nVšechna data na tomto USB flash disku budou ztracena."

#: ../tails_installer/gui.py:774
#, python-format
msgid "%(parent_size)s %(vendor)s %(model)s device (%(device)s)"
msgstr "%(parent_size)s %(vendor)s %(model)s device (%(device)s)"

#: ../tails_installer/gui.py:782
msgid ""
"\n"
"\n"
"The persistent storage on this USB stick will be preserved."
msgstr "\n\nTrvalé úložiště na tomto USB flash disku bude zachováno."

#: ../tails_installer/gui.py:783
#, python-format
msgid "%(description)s%(persistence_message)s"
msgstr "%(description)s %(persistence_message)s"

#: ../tails_installer/gui.py:826
msgid "Download complete!"
msgstr "Stahování dokončeno!"

#: ../tails_installer/gui.py:830
msgid "Download failed: "
msgstr "Stahování selhalo: "

#: ../tails_installer/gui.py:831
msgid "You can try again to resume your download"
msgstr "Zkuste znovu zahájit stahování"

#: ../tails_installer/gui.py:839
msgid ""
"The selected file is unreadable. Please fix its permissions or select "
"another file."
msgstr "Vybraný soubor je nečitelný. Opravte jeho oprávnění nebo vyberte jiný soubor."

#: ../tails_installer/gui.py:845
msgid ""
"Unable to use the selected file.  You may have better luck if you move your "
"ISO to the root of your drive (ie: C:\\)"
msgstr "Nemohu použít vybraný soubor. Zkuste přesunout ISO do kořenového adresáře vašeho disku (např. C:\\)"

#: ../tails_installer/gui.py:851
#, python-format
msgid "%(filename)s selected"
msgstr "%(filename)s zvolen"

#: ../tails_installer/source.py:28
msgid "Unable to find LiveOS on ISO"
msgstr "Nelze na ISO najít LiveOS"

#: ../tails_installer/source.py:34
#, python-format
msgid "Could not guess underlying block device: %s"
msgstr "Nelze odhadnout základní blokové zařízení: %s"

#: ../tails_installer/source.py:49
#, python-format
msgid ""
"There was a problem executing `%s`.\n"
"%s\n"
"%s"
msgstr "Nastal problém při spouštění `%s`.\n%s\n%s"

#: ../tails_installer/source.py:63
#, python-format
msgid "'%s' does not exist"
msgstr "'%s' neexistuje"

#: ../tails_installer/source.py:65
#, python-format
msgid "'%s' is not a directory"
msgstr "'%s' není složka"

#: ../tails_installer/source.py:75
#, python-format
msgid "Skipping '%(filename)s'"
msgstr "Přeskočit '%(filename)s'"

#: ../tails_installer/utils.py:44
#, python-format
msgid ""
"There was a problem executing `%s`.%s\n"
"%s"
msgstr "Nastal problém při spuštění `%s`.%s\n%s"

#: ../tails_installer/utils.py:124
msgid "Could not open device for writing."
msgstr "Nelze otevřít zařízení pro zápis."

#: ../data/tails-installer.ui.h:4
msgid "Select a distribution to download:"
msgstr "Vyberte distribuci ke stažení:"

#: ../data/tails-installer.ui.h:5
msgid "Target USB stick:"
msgstr "Cílový USB flash disk:"

#: ../data/tails-installer.ui.h:6
msgid "Reinstall (delete all data)"
msgstr "Přeinstalovat (smazat všechna data)"
